x-pack/plugins/task_manager/server/monitoring/workload_statistics.ts

```diff
@@ -147,8 +147,19 @@ export function createWorkloadAggregator(
             missing: { field: 'task.schedule' },
           },
           ownerIds: {
-            cardinality: {
-              field: 'task.ownerId',
+            filter: {
+              range: {
+                'task.startedAt': {
+                  gte: 'now-1w/w',
```

Comment:
Is `1w` arbitrary, or is that some meaningful amount of time in relation to how the system works? Is it worthwhile exposing this as a configuration option for any reason? I'm not sure but just want to make sure I understand why `1w` and if we think that will work well for everyone.

Reply:
It is kind of arbitrary and I'm open to feedback! I don't know that I necessarily see the value of exposing it as a config value since `ownerId` cardinality is really just used internally for this capacity estimation.

Reply:
I honestly don't know a good value here. I have no issues with `1w` but I'm starting to think we could/should leverage stack monitoring to get these counts (assuming monitoring is enabled which I think is a safe bet for folks interested in these metrics in the first place). SM could expose a method in their start contract that gives us the "active uuids" for the cluster and we perform a `terms` query with that to ensure we only are looking at tasks from active Kibanas. Anyways, that's a separate discussion but feels like maybe a good direction for the future.